### PR TITLE
Fix -Wformat-truncation warnings in cbin.c for #10338

### PR DESCRIPTION
**Pull request by kazarmy**

### CODE REVIEW

File: libr/core/cbin.c [hunk 1]
```diff
@@ -2904,7 +2904,7 @@ static void bin_pe_versioninfo(RCore *r, int mode) {
 		} else {
 			r_cons_printf ("# VS_FIXEDFILEINFO\n\n");
 		}
-		char path_fixedfileinfo[256] = R_EMPTY;
+		char path_fixedfileinfo[256 + 16] = R_EMPTY;
```
Comment (radare) [inline libr/core/cbin.c]: just use r_str_newf, dont use hardcoded non-null-terminated-stack-based-string-holders please :P

Comment (kazarmy) [reply libr/core/cbin.c]: i'm just the warning-fixer 😅 but ok

**Comment by radare** [comment]
i fixed that already in master

**Comment by kazarmy** [comment]
thanks!

**Pull request merged**